controllers/pwas/crud.js

```diff
@@ -136,7 +136,8 @@ router.post('/:pwa/edit', (req, res, next) => {
 router.get('/:pwa', (req, res, next) => {
   pwaModel.find(req.params.pwa, (err, entity) => {
     if (err) {
-      return next(err);
+      // Not really an error: the pwa wasn't found in the db. Fall through to 404 page.
```

Comment:
Should the find method invoke the callback(null, null), so both err and entity are null? Maybe it's more clear in the code than testing an error that's not exactly an error.